recipes/sed-plume/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set name = "sed-plume" %}
+{% set version = "0.2.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/mcflugen/plume/archive/v{{ version }}.tar.gz
+  sha256: ec764a27d67737353336396d6d460a08f391ae2745a17aa088421e6f28164c30
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  skip: True  # [py<38]
+  entry_points:
+    - plume = plume.cli:plume
+
+requirements:
+  build:
+    # If your project compiles code (such as a C extension) then add the required compilers as separate entries here.
+    # Compilers are named 'c', 'cxx' and 'fortran'.
+    - {{ compiler('c') }}
+  host:
+    - python
+    - pip
+    - cython
+    - cythongsl
+    - numpy
+    - gsl
+  run:
+    - python
+    - gsl
+    - libopenblas
```

Comment:
```suggestion
```
BLAS is not a python library or a binary. It should not be in the run requirements without also being in the host section. If your package links against BLAS, please follow the BLAS directions in the docs.

https://conda-forge.org/docs/maintainer/knowledge_base.html?highlight=blas#blas

But also, does this package link to BLAS? The linker doesn't detect links to BLAS on any platform.
https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=529045&view=logs&j=e35d4f76-8ff2-5536-d795-df91e63eb9f7&t=fa7b4b17-b6ff-5c9c-8cfc-15f888c92310&l=4853

Reply:
@carterbox I had added *libopenblas* as a requirement to solve a linker error I was getting but that error seems to have gone away!

Reply:
Looks like your project includes the following headers

```
#include "gsl/gsl_cblas.h"
#include "gsl/gsl_blas_types.h"
#include "gsl/gsl_blas.h"
```
And the logs show that `centerline.cpython-39-x86_64-linux-gnu.so` links to libopenblas.

However, I don't know enough about your package to tell whether this link to libopenblas is because there are actually libopenblas symbols included in those headers or if there is just a mistake in your build process that accidentally links to libopenblas when it is not needed.

Reply:
Interesting. I also would have thought that if *libopenblas* was required it would have been listed as a dependency for *gsl*.